test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresUnreferencedCodeCapability.cs

```diff
@@ -56,7 +56,7 @@ public static void Main ()
 			TestRequiresInDynamicDependency ();
 		}
 
-		[ExpectedWarning ("IL2026", "--RequiresWithMessageOnly--")]
+		[ExpectedWarning ("IL2026", new string[] { "'Mono.Linker.Tests.Cases.RequiresCapability.RequiresUnreferencedCodeCapability.RequiresWithMessageOnly()'", "Message for --RequiresWithMessageOnly--." })]
```

Comment:
Why add `new string[]`?

Reply:
Ok, I changed the code to not support array creation, the code will throw in GetStringFromExpr if it ever tries to be used